modules/partition-replicator/src/integrationTest/java/org/apache/ignite/internal/partition/replicator/ItZoneDataReplicationTest.java

```diff
@@ -404,6 +420,99 @@ void testDataRebalance(boolean truncateRaftLog) throws Exception {
         assertThat(kvView2.getAll(null, data2.keySet()), is(data2));
     }
 
+    @ParameterizedTest
+    @ValueSource(booleans = {true, false})
+    void txFinishCommandGetsReplicated(boolean commit) throws Exception {
```

Comment:
I would suggest to extract this test in a separate class, because it will become super huge if we will be adding all commands here. But I don't insist.